### ISSUE TITLE
path show doesn't show destination anymore

### ISSUE THREAD

**Issue by jopietsch** [netenglabs/suzieq#165]
**Describe the bug**
used to show
```
(suzieq-ITL68y-d-py3.7) jpiet@a2:~/suzieq$ python3 suzieq/cli/suzieq-cli -c ~/configs/multidc.yml path show --dest=172.16.4.104 --src=172.16.1.101  --namespace=ospf-single
   pathid  hopCount    namespace   hostname     iif      vrf  overlay mtuMatch   mtu               timestamp
0       1         0  ospf-single  server101    eth1  default    False      NaN  1500 2020-06-16 20:43:18.629
1       1         1  ospf-single     leaf01  vlan10  default    False     True  1500 2020-06-16 20:43:17.009
2       1         2  ospf-single    spine01    swp1  default    False     True  1500 2020-06-16 20:43:17.009
3       1         3  ospf-single     leaf04    swp1  default    False     True  1500 2020-06-16 20:43:16.979
4       1         4  ospf-single  server104    eth1  default    False      NaN  1500 2020-06-16 20:43:19.557
5       2         0  ospf-single  server101    eth1  default    False      NaN  1500 2020-06-16 20:43:18.629
6       2         1  ospf-single     leaf01  vlan10  default    False     True  1500 2020-06-16 20:43:17.009
7       2         2  ospf-single    spine02    swp1  default    False     True  1500 2020-06-16 20:43:17.009
8       2         3  ospf-single     leaf04    swp2  default    False     True  1500 2020-06-16 20:43:17.009
9       2         4  ospf-single  server104    eth1  default    False      NaN  1500 2020-06-16 20:43:19.557
```
what it shows now

```
(suzieq-QZqWZG2D-py3.8) jpietsch@jpietsch-server:~/suzieq$ python3 suzieq/cli/suzieq-cli -c ~/configs/multidc.yml path show --dest=172.16.4.104 --src=172.16.1.101  --namespace=ospf-single
   pathid  hopCount    namespace   hostname     iif      vrf  overlay mtuMatch   mtu               timestamp
0       1         0  ospf-single  server101    eth1  default    False      NaN  1500 2020-06-17 05:20:20.670
1       1         1  ospf-single     leaf01  vlan10  default    False     True  1500 2020-06-17 05:20:18.583
2       1         2  ospf-single    spine01    swp1  default    False     True  1500 2020-06-17 05:20:18.441
3       1         3  ospf-single     leaf04    swp1  default    False     True  1500 2020-06-17 05:20:18.441
4       2         0  ospf-single  server101    eth1  default    False      NaN  1500 2020-06-17 05:20:20.670
5       2         1  ospf-single     leaf01  vlan10  default    False     True  1500 2020-06-17 05:20:18.583
6       2         2  ospf-single    spine02    swp1  default    False     True  1500 2020-06-17 05:20:18.441
7       2         3  ospf-single     leaf04    swp2  default    False     True  1500 2020-06-17 05:20:18.435
```

**Comment by jopietsch** [comment]
this is when the last hop doesn't have lldp. i still think we should do something here

**Comment by ddutt** [comment]
Fixed with commit  80175595526cf5bf8d31bcf881b8f27830d8c1c9
